es/index.js

```diff
@@ -16,4 +16,10 @@ if (typeof self !== 'undefined') {
 }
 
 var result = ponyfill(root);
+
+root.Symbol = root.Symbol || function Symbol(description) {
+  return '@@' + description;
+};
+root.Symbol.observable = result;
```

Comment:
This will still fail if `Symbol` (a primordial) is frozen.

Reply:
There are two entry points. This is the polyfill (mutates globals) and the other is a ponyfill (the other module) is the subset that exports its API and does not mutate globals.